README.md

```diff
@@ -1,11 +1,56 @@
 It is an open organization where people are judged solely on the quality of their contributions.
-
 Are you interested in volunteering your time to help the development of Ethereum Classic? Post in the issues section, tell us about yourself and we will try to help you find a place you can contribute to Ethereum Classic development.
+If you are interested in our community organization, you can find more information about volunteering with other community organizers here: https://github.com/ethereumclassic
+
+## Volunteers' GPG Keys
+Anyone is free to submit pull requests, issues, and general code reviews as an outside contributor.
+For membership into the organization allows contributors a higher level of access to the code contained in this repository and, as such, requires additional security considerations.   
+For membership:
+-	Be an active contributor to a repository at least once every 3 months, or have a substantial body of work invested within the project
+-	Have 2fa enabled
+-	Submit a PR with a valid GPG key to the volunteer list
+https://github.com/ethereumclassic/volunteer/tree/master/Volunteer-Public-Keys - GPG keys of volunteers on the Ethereum Classic project
+### Administrator / owner access
+Owner access allows users the highest level of access to the repository and should only be granted under overwhelmingly obvious need. 
+NO OWNER SHALL ADD OR REMOVE ANY OWNER WITHOUT EXPRESS WRITTEN CONSENT OF ALL OTHER OWNERS. 
+### Current Owners
+-	Cody Burns
+-	Elaine Ou
+-	WhatisGravity
 
-If you are interested in community organization, you can find more information about volunteering with other community organizers here: https://github.com/ethereumclassic
+## Structure
+
+This “organization” is designed to operate development in a similar fashion to the Angel
+volunteer system found at CCC events. Existing volunteers will do their best to
+help new volunteers, but there is no strict leadership hierarchy.
+Instead of a chain of command, we operate on a merit system where you are judged only on your code & discussion contributions, not your age, race, gender, sex, nationality or anything else, just on what you contribute. You do not even need to own any Ethereum classic, if you have a good idea or a well reasoned contribution, that is all that matters.
+If you see anything that needs to be fixed and you are capable, please feel free to fix it, create a local fork, make the change and submit a pull request.
+Everyone is currently working in a volunteer capacity, the fastest way to enact changes is to create pull requests.
+If you would like to make fundamental changes to the system, please consult with your peers, have them review your code, and make sure we have consensus. If the issue is contentious we should work to include the input of the wider community.
+
+## Coding conventions
+  * We indent using two spaces (soft tabs)
```

Comment:
This isn't wrong in the PR but rather a note. Some of these issues can be prevented if we all agree to use a code formatter (assuming on exists for the language) and encourage others to do so as well (or at least have a linter configuration in the repo)

Reply:
Agreed.  @phyro if you are free maybe make an Issue regarding this and once things settle down, we can really start looking to formalizing contributions (styling, ci linting, builds, etc).  I can do so later when I am attached to the console again.

Reply:
see: https://github.com/ethereumclassic/volunteer/issues/2